Reqres/Reqres.swift

```diff
@@ -60,7 +61,10 @@ open class Reqres: URLProtocol, URLSessionDelegate {
         URLProtocol.setProperty(true, forKey: ReqresRequestHandledKey, in: newRequest!)
         URLProtocol.setProperty(Date(), forKey: ReqresRequestTimeKey, in: newRequest!)
 
-        let session = URLSession(configuration: .default, delegate: Reqres.sessionDelegate ?? self, delegateQueue: nil)
+        let config = URLSessionConfiguration.default
+        config.protocolClasses = config.protocolClasses!.filter {$0 != Reqres.self}
```

Comment:
It would be nice to get rid of the exclamation to be completely safe (even though you are sure there is an array of `protocolClasses`).

Also our internal (🤷‍♂️)  guides say that parenthesis should contain spaces inside
```swift
filter { $0 != Reqres.self }
```